pendulum_control/test/execute_with_delay.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2015 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import subprocess
+import sys
+import time
+
+
+def main():
+    if len(sys.argv) < 3:
+        print("usage: execute_with_delay.py <delay (ms)> <executable> [args ...]")
```

Comment:
Errors should go to `stderr`. Why not use `argparse` it will handle the error reporting for you?